pkg/verify/verify.go

```diff
@@ -151,16 +72,28 @@ type ClientBuilder interface {
 	HTTPClient() (*http.Client, error)
 }
 
-// simpleClientBuilder implements the ClientBuilder interface and should be used for testing.
+// DefaultClient uses the default http.Client for accessing signatures.
+var DefaultClient = simpleClientBuilder{}
+
+// simpleClientBuilder implements the ClientBuilder interface and may be used for testing.
```

Comment:
why add DefaultClient to package api?

Reply:
So test packages in others can use it.  I had to use it in 2-3 packages that wouldn't be in verify (and might not be in library go).  It's a standard implementation.